lib/util.js

```diff
@@ -335,6 +335,25 @@ module.exports.normalizePath = function(oldPath, relativeTo) {
   return normalized;
 }
 
+module.exports.getRemoteIframeTemplate = function() {
+  let defaultRemoteAppTemplate = fs.readFileSync(path.join(__dirname,'default-remote-app-template.html'),"utf-8");
```

Comment:
this should be put as a const outside the function call, since you only need to load it once in order to use it multiple times, right?

Reply:
even before it was a IFFE, but just moved it outside